apps/meteor/package.json

```diff
@@ -195,12 +195,12 @@
 		"@rocket.chat/emitter": "~0.31.11",
 		"@rocket.chat/forked-matrix-appservice-bridge": "^4.0.1",
 		"@rocket.chat/forked-matrix-bot-sdk": "^0.6.0-beta.2",
-		"@rocket.chat/fuselage": "~0.31.11",
+		"@rocket.chat/fuselage": "next",
```

Comment:
@MartinSchoeler latest didnt work. As pointed by @murtaza98 , the tagging for icons and fuselage repo was not latest but "next". So this worked.

Though I have a question. I am sure we can not merge this file with _**"@rocket.chat/fuselage": "next",**_
If so, what version should we put here while merging the code with develop?